views/result/account.validation.pending.html

```diff
@@ -0,0 +1,21 @@
+{% set html %}
+    <h3 class="govuk-notification-banner__heading">
+        The validation of your document '{{accountValidationResult.fileName}}' is in progress
+    </h3>
+    <p class="govuk-body">The is in progress. This page will automatically update when it is complete.</p>
```

Comment:
This sentence is incomplete- "The is in progress" could you please fix this ? It should be validation in progress.

Reply:
I'll add a commit to address this.